### ISSUE TITLE
SQL file missing

### ISSUE THREAD

**Issue by amitsavla** [sudogem/cebufreelancer#2]
SQL file is missing

**Comment by sudogem** [comment]
@amitsavla Yeah, right now the sql schema is on the remote DB server which I haven't dump locally. I'm looking for a better ways on how to create a database migration in CodeIgniter.
Something like how the Rails DB migration works. Do you have any idea?

**Comment by amitsavla** [comment]
@sudogem - hi , if you can give me basic schema so i can try to resolve
